gateway/api/use_cases/dependencies_versions.py

```diff
@@ -0,0 +1,22 @@
+"""Authentication use case to manage the authentication process in the api."""
+
+import logging
+from typing import Optional
+from django.contrib.auth.models import AbstractUser
+
+from api.utils import create_dynamic_dependencies_whitelist
+
+
+logger = logging.getLogger("gateway.use_cases.authentication")
+
+
+class AvailableDynamicDependenciesUseCase:  # pylint: disable=too-few-public-methods
```

Comment:
I would match more or less the name of the class with the file, WDYT @korgan00 ? My proposal: `AvailableDependenciesVersions`.